### PR TITLE
My very first Pull Request!

### PR DESCRIPTION
**Pull request by noVaSon**
Dear Frederic,

this is my first attempt to transmit my changes, just want to see it it is the right way to do this.
I am a little confused by git an working with branches - there is still a lot of useless merging.
I'll adapt my workflow to working on different things in different branches from now on.

**It contains only pathes of the original FSE master branch. No new features here.**

Details: 
I changed the template string "/fse/" in the /src/constants file because I had misunderstood it first and passed in the root of my repo. So I could not save an load sessions.

Cheers!

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const DEFAULT_DESCRIPTOR = 'lowlevel.mfcc.mean';
 export const PERFORM_QUERY_AT_MOUNT = false;
 
 // backend and login urls
-const BACKEND_APPLICATION_ROOT = '/fse';  // must match with backend's application root
+const BACKEND_APPLICATION_ROOT = '/fse/backend' // -> must match with backend's application root
```
Comment (ffont) [inline src/constants.js]: Don't understand why you made this change here, I think we should keep `/fse` or move this to some option that can be configured elsewhere. The deployed version of FSE uses /fse`.

**Comment by ffont** [comment]
Thanks for the contribution!
I looks fine (although I have not tested it yet), but I think the `BACKEND_APPLICATION_ROOT` should be changed to what it was. Why leaving the `/fse/backend` path?

File: src/constants.js (hunk 1 above)
Comment (noVaSon) [reply src/constants.js]: I can not remember - had some trouble with saving and loading in the beginning and changed some paths. Just ignore if it works that way.

Comment (ffont) [reply src/constants.js]: I'd say make sure that you can make it work reverting that change and commit again in this PR. Then we can merge.

**Comment by ffont** [comment]
Closing this as is included in #37

**Pull request closed without merging**